extensions/amp-image-lightbox/0.1/amp-image-lightbox.js

```diff
@@ -223,10 +235,22 @@ export class ImageViewer {
     this.sourceWidth_ = sourceElement./*OK*/offsetWidth;
     this.sourceHeight_ = sourceElement./*OK*/offsetHeight;
     this.srcset_ = srcsetFromElement(sourceElement);
+    Object.keys(this.captions_).forEach(key => {
+      if (key == 'aria-describedby') {
```

Comment:
I would turn `aria-describedby` around and have `AmpImageLightbox` handle it instead of `ImageViewer` since it is not really a cloned attribute like `aria-label` it.

Essentially, what I am recommending is to have 
`AmpImageLightbox`  call `getImage()` on the `ImageViewer` and set the `aria-describedby` on `init_` to the id it has and remove it on `reset_` (so `ImageViewer` has no idea of the id of the caption which is part of the `AmpImageLightbox` and `captionElement` can go back to being private.